pyomo/contrib/doe/doe.py

```diff
@@ -1183,7 +890,7 @@ def trace_calc(m):
             """
             Calculate FIM elements. Can scale each element with 1000 for performance
             """
-            return m.trace == sum(m.FIM[j,j] for j in m.para_set)
+            return m.trace == sum(m.fim[j,j] for j in m.para_set)
```

Comment:
Should m.para_set be changed? If yes, this implies this function is not tested. We should add a simple unit test.

Reply:
added a test to use trace as objective function